adapters/repos/db/shard.go

```diff
@@ -95,7 +89,7 @@ type ShardLike interface {
 	ObjectDigestsByTokenRange(ctx context.Context, initialToken, finalToken uint64, limit int) (objs []replica.RepairResponse, lastTokenRead uint64, err error)
 	ID() string // Get the shard id
 	drop() error
-	HaltForTransfer(ctx context.Context) error
+	HaltForTransfer(offloading bool, ctx context.Context) error
```

Comment:
It's a best practice to [have `ctx context.Context` be the first argument to the function](https://docs.datadoghq.com/code_analysis/static_analysis_rules/go-best-practices/context-first-argument/)